From 6774710b4c7b6343951e9eaa35a1516abb975fd5 Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Tue, 19 Nov 2013 12:14:55 -0500
Subject: [PATCH 133/159] LSI AXM55XX SIM: Add VGIC and virtual timer entries
 in DTS file

git.yoctoproject.org/git/linux-yocto-3.10
commit dc7a07b6f3aa4ec8dca516c78b5538a11a4f777f standard/axxia/base.

Add vgic and virtual timer entries in axm55xxsim.dts file.

Signed-off-by: David Mercado <david.mercado@windriver.com>
---
 arch/arm/boot/dts/axm55xxsim.dts |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/axm55xxsim.dts b/arch/arm/boot/dts/axm55xxsim.dts
index 1c6b413..6cc60ab 100644
--- a/arch/arm/boot/dts/axm55xxsim.dts
+++ b/arch/arm/boot/dts/axm55xxsim.dts
@@ -166,6 +166,8 @@
 		interrupt-controller;
 		reg = <0x20 0x01001000 0 0x1000>,  /* gic dist base */
 		      <0x20 0x01002000 0 0x100>,   /* gic cpu base */
+		      <0x20 0x01004000 0 0x2000>,  /* vgic control */
+		      <0x20 0x01006000 0 0x2000>,  /* vgic cpu base */
 		      <0x20 0x10030000 0 0x100>,   /* axm IPI mask reg base */
 		      <0x20 0x10040000 0 0x20000>; /* axm IPI send reg base */
 	};
@@ -173,7 +175,9 @@
 	timer {
 		compatible = "arm,armv7-timer";
 		interrupts = <1 13 0xf08>,
-			     <1 14 0xf08>;
+			     <1 14 0xf08>,
+			     <1 11 0xf08>,
+			     <1 10 0xf08>;
 	};
 
 
-- 
1.7.5.4

